addons/website_payment/views/donation_templates.xml

```diff
@@ -73,13 +73,13 @@
                                             </t>
                                             <t t-if="donation_layout">
                                                 <div class="custom-control custom-radio my-2">
-                                                    <input class="custom-control-input" type="radio" id="other_amount" name="amount" t-att-value="amount"/>
+                                                    <input class="fee_impact custom-control-input" type="radio" id="other_amount" name="amount" t-att-value="amount"/>
                                                     <label class="custom-control-label mt-0" for="other_amount">Other Amount</label>
                                                 </div>
                                             </t>
                                         </t>
-                                        <input type="number" class="form-control o_amount_input d-none" t-att-min="donation_options.get('minimumAmount')"
-                                            t-att-max="donation_options.get('maximumAmount')" t-att-value="amount"/>
+                                        <input type="number" class="fee_impact form-control o_amount_input d-none" t-att-min="donation_options.get('minimumAmount')"
+                                            t-att-max="donation_options.get('maximumAmount')" t-att-value="amount" id="other_amount_value"/>
```

Comment:
@bso-odoo Do you still need this id (#other_amount_value) ?

Reply:
It seems it is actually not used.